layouts/_default/baseof.html

```diff
@@ -1,29 +1,23 @@
-{{ $dropdown   := site.Menus.dropdown }}
-{{ $currentUrl := .RelPermalink }}
-{{ $lang       := site.LanguageCode }}
```

Comment:
This is a single use variable: inlined use of `site.LanguageCode` is just as clear in terms of intent as `$lang`, which some might actually consider more ambiguous.